projects/admin/src/app/record/detail-view/organisation-detail-view/organisation-detail-view.component.ts

```diff
@@ -44,7 +44,7 @@ export class OrganisationDetailViewComponent implements DetailRecord {
    * @return True | False depending of the current logged user role
    */
   get isSystemLibrarian(): boolean {
-    return this._userService.hasRole('system_librarian');
+    return this._userService.user.isLibrarian;
```

Comment:
`isLirbarian` methdod check if user has the 'librarian' role, not the system_librarian role